python/lsst/source/injection/inject_coadd.py

```diff
@@ -72,6 +128,13 @@ class CoaddInjectTask(BaseInjectTask):
     _DefaultName = "coaddInjectTask"
     ConfigClass = CoaddInjectConfig
 
+    def run(self, injection_catalogs, input_exposure, psf, photo_calib, wcs):
+        self.log.info("Fitting flux vs. variance in each pixel.")
+        variance_scale = self.get_variance_scale(input_exposure)
+        self.log.info("Variance scale factor: %.6f", variance_scale)
```

Comment:
Can these log messages be moved into the `get_variance_scale` method, for clarity when calling it?